### PR TITLE
TCVP-1245 env and version header

### PR DESCRIPTION
**Pull request by lfdgov**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ import { FileUtilsService } from '@shared/services/file-utils.service';
 import { Field, OcrViolationTicket, TicketsService, ViolationTicket } from 'app/api';
 import { AppRoutes } from 'app/app.routes';
 import { NgProgressRef } from 'ngx-progressbar';
-import { BehaviorSubject, catchError, map, Observable } from 'rxjs';
+import { BehaviorSubject, Observable } from 'rxjs';
```
Comment (lfdgov) [inline src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts:14]: fixes a compile error in citizen portal

File: src/frontend/staff-portal/src/app/config/config.service.ts [hunk 2]
```diff
@@ -107,13 +107,7 @@ export class ConfigService implements IConfigService {
...
   public load(): Observable<Configuration> {
     if (!this.configuration) {
-      return this.appConfigService.loadAppConfig().pipe(
-        switchMap(() => {
-          return this.getConfiguration().pipe(
-            map((config: Configuration) => (this.configuration = config))
-          );
-        })
-      );
+      return this.appConfigService.loadAppConfig()
```
Comment (lfdgov) [inline src/frontend/staff-portal/src/app/config/config.service.ts:110]: consistent with config loading in citizen portal

File: src/frontend/staff-portal/src/app/core/core.module.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { ErrorHandlerService } from './services/error-handler.service';
 
 export function initConfig(config: ConfigService) {
   return () => {
-    // return config.load();
+    return config.load();
```
Comment (lfdgov) [inline src/frontend/staff-portal/src/app/core/core.module.ts:10]: so environment and version are loaded, matches config loading in citizen portal

**Review by Sharkww — APPROVED** [review]

File: src/frontend/staff-portal/src/app/services/app-config.service.ts [hunk 4]
```diff
@@ -9,13 +9,6 @@ export interface IAppConfig {
   version: string;
   useMockServices: boolean;
   apiBaseUrl: string;
-  understandYourTicketLink: string;
```
Comment (lfdgov) [inline src/frontend/staff-portal/src/app/services/app-config.service.ts:12]: cleaning up unused code copied over from citizen portal

File: src/frontend/staff-portal/src/app/shared/components/header/header.component.html [hunk 5]
```diff
@@ -8,7 +8,11 @@
            rel="noopener">
           <img src="/assets/gov_bc_logo.jpg"
                alt="Government of BC Logo" />
-        </a>| Ticket Resolution  Management
+        </a>| Ticket Resolution  Management 
+        <span class="badge badge-pill badge-light header"
+            *ngIf="environment !== 'prod'">
+        {{ environment }} v{{ version }}
+      </span>
```
Comment (lfdgov) [inline src/frontend/staff-portal/src/app/shared/components/header/header.component.html:15]: display of environment and version in different location than citizen portal, due to overlap with Sign in / Sign out IDIR buttons.  cleared with Business Analyst

File: src/frontend/staff-portal/src/app/shared/components/header/header.component.scss [hunk 6]
```diff
@@ -23,10 +23,6 @@
     cursor: pointer;
   }
 
-  .fullName {
```
Comment (lfdgov) [inline src/frontend/staff-portal/src/app/shared/components/header/header.component.scss:26]: deleting unused code

**Pull request merged**